tests/e2e/support/api/graph/spaces.ts

```diff
@@ -12,38 +12,27 @@ interface DrivesResponse {
   value: Space[]
 }
 
-export const getPersonalSpaceId = async ({ user }: { user: User }): Promise<string> => {
-  const response = await request({
-    method: 'GET',
-    path: join('graph', 'v1.0', 'me', 'drives', "?$filter=driveType eq 'personal'"),
-    user: user
-  })
-
-  checkResponseStatus(response, 'Failed while geting personal space')
-
-  const resBody = (await response.json()) as DrivesResponse
-  return resBody.value[0].id
-}
-
-export const getSpaceIdBySpaceName = async ({
+export const getSpacesIdBySpacesName = async ({
   user,
   spaceType,
   spaceName
 }: {
   user: User
   spaceType: string
-  spaceName: string
+  spaceName?: string
 }): Promise<string> => {
   const response = await request({
     method: 'GET',
-    path: join('graph', 'v1.0', 'me', 'drives', `?$filter=driveType eq '${spaceType}'`),
+    path: join('graph', 'v1.0', 'me', 'drives'),
     user: user
   })
   checkResponseStatus(response, 'Failed while fetching spaces')
   // search for the space with the space name
   const resBody = (await response.json()) as DrivesResponse
   for (const spaceProject of resBody.value) {
-    if (spaceProject.name === spaceName) {
+    if (spaceProject.driveType === spaceType) {
+      return spaceProject.id
+    } else if (spaceProject.driveType === spaceType && spaceProject.name === spaceName) {
```

Comment:
here, the else block will never be true. because if `spaceProject.driveType === spaceType` is false then else will also be false